<a href="https://colab.research.google.com/github/Izaak104/healthcare-visuals/blob/main/data_visuals.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

# Step 1: Loading Packages

In [ ]:
#load libraries packages for code reading on data transformation
import pandas as pd
import numpy as np


In [ ]:
import seaborn as sns
import matplotlib.pyplot as plt
import plotly.express as px

sns.set_theme(style="whitegrid")

# Step 2: Uploading data


In [ ]:
dataframe = pd.read_csv('https://raw.githubusercontent.com/hantswilliams/AHI_Microcourse_Visualization/main/Data/Georgia_COVID/Georgia_COVID-19_Case_Data.csv')

In [ ]:
dataframe

OBJECTID   COUNTY  CNTY_FIPS               DATESTAMP  C_New  C_Cum  \
0             1  BALDWIN       1003  2021/04/13 03:59:59+00      4   3795   
1             2  UNKNOWN          1  2021/02/04 05:00:00+00      4   2158   
2             3  BALDWIN       1003  2021/04/14 03:59:59+00      2   3797   
3             4  UNKNOWN          1  2021/02/05 05:00:00+00      1   2159   
4             5  BALDWIN       1003  2021/04/15 03:59:59+00      0   3797   
...         ...      ...        ...                     ...    ...    ...   
91687     91688    WAYNE      49055  2021/07/15 03:59:59+00      0   2809   
91688     91689    WAYNE      49055  2021/07/16 03:59:59+00      0   2809   
91689     91690    WAYNE      49055  2021/07/17 03:59:59+00      0   2809   
91690     91691    WAYNE      49055  2021/07/18 03:59:59+00      0   2809   
91691     91692    WAYNE      49055  2021/07/19 03:59:59+00      0   2809   

       D_New  D_Cum  H_New  H_Cum  ...  C_His  C_NonHis  C_EthUnk  \
0          0    115      0    318  ...    106      2897       792   
1          0      4      0     36  ...     80       155      1923   
2          0    115      0    318  ...    106      2898       793   
3          0      4      0     36  ...     80       156      1923   
4          0    115      0    318  ...    106      2898       793   
...      ...    ...    ...    ...  ...    ...       ...       ...   
91687      0     81      0    281  ...    108      2070       631   
91688      0     81      0    281  ...    108      2070       631   
91689      0     81      0    281  ...    108      2070       631   
91690      0     81      0    281  ...    108      2070       631   
91691      0     81      0    281  ...    108      2070       631   

                                     GlobalID  C_NEW_PERCT_CHG  \
0      {6D1CBB20-5FCF-4FA3-9112-504CF3A93EE8}              NaN   
1      {925F3292-9C8D-420F-9FE5-97E82BC7F8E8}        -0.428571   
2      {1F195651-C20F-4B60-98B0-D0481D40F263}              NaN   
3      {C614DA89-4EAB-4807-9FDE-7661AEFCC144}              NaN   
4      {EBFFE6DD-9C7F-44F1-B19A-AC69E6EB4093}              NaN   
...                                       ...              ...   
91687  {D678A5A4-8531-4A06-B71A-A757A4933957}              NaN   
91688  {6F4E3A27-D6A2-422F-B7CA-16595DA36061}              NaN   
91689  {CE952CC5-B75F-45E7-BC12-954B1EE71F31}              NaN   
91690  {D07AF78B-C535-4F99-8E04-AED183809453}              NaN   
91691  {42B36DB7-6C0D-4A0D-AE78-E269A191E20E}              NaN   

       D_NEW_PERCT_CHG  C_NEW_7D_MEAN  D_NEW_7D_MEAN  C_NEW_7D_PERCT_CHG  \
0                  NaN       1.857143            0.0                 NaN   
1                  NaN       2.714286            0.0                 NaN   
2                  NaN       1.857143            0.0                 NaN   
3                  NaN       2.857143            0.0                 NaN   
4                  NaN       1.857143            0.0                 NaN   
...                ...            ...            ...                 ...   
91687              NaN       0.142857            0.0                 NaN   
91688              NaN       0.000000            0.0                 NaN   
91689              NaN       0.000000            0.0                 NaN   
91690              NaN       0.000000            0.0                 NaN   
91691              NaN       0.000000            0.0                 NaN   

       D_NEW_7D_PERCT_CHG  
0                     NaN  
1                     NaN  
2                     NaN  
3                     NaN  
4                     NaN  
...                   ...  
91687                 NaN  
91688                 NaN  
91689                 NaN  
91690                 NaN  
91691                 NaN  

[91692 rows x 45 columns]

In [ ]:
len(dataframe)

91692

In [ ]:
dataframe.shape

(91692, 45)

# Step 3: Describing the variables

In [ ]:
dataframe.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 91692 entries, 0 to 91691
Data columns (total 45 columns):
 #   Column              Non-Null Count  Dtype  
---  ------              --------------  -----  
 0   OBJECTID            91692 non-null  int64  
 1   COUNTY              91692 non-null  object 
 2   CNTY_FIPS           91692 non-null  int64  
 3   DATESTAMP           91692 non-null  object 
 4   C_New               91692 non-null  int64  
 5   C_Cum               91692 non-null  int64  
 6   D_New               91692 non-null  int64  
 7   D_Cum               91692 non-null  int64  
 8   H_New               91692 non-null  int64  
 9   H_Cum               91692 non-null  int64  
 10  C_Rate              91692 non-null  int64  
 11  C_Female            91692 non-null  int64  
 12  C_Male              91692 non-null  int64  
 13  C_SexUnkn           91692 non-null  int64  
 14  C_UCon_Yes          91692 non-null  int64  
 15  C_UCon_No           91692 non-null  int64  
 16  C_UC

In [ ]:
list(dataframe)

['OBJECTID',
 'COUNTY',
 'CNTY_FIPS',
 'DATESTAMP',
 'C_New',
 'C_Cum',
 'D_New',
 'D_Cum',
 'H_New',
 'H_Cum',
 'C_Rate',
 'C_Female',
 'C_Male',
 'C_SexUnkn',
 'C_UCon_Yes',
 'C_UCon_No',
 'C_UCon_Unk',
 'C_Age_0_4',
 'C_Age_5_14',
 'C_Age_15_24',
 'C_Age_25_34',
 'C_Age_35_44',
 'C_Age_45_54',
 'C_Age_55_64',
 'C_Age_65_74',
 'C_Age_75_84',
 'C_Age_85plus',
 'C_Age_Unkn',
 'C_Age_0',
 'C_Age_20',
 'C_RaceWh',
 'C_RaceBl',
 'C_RaceAs',
 'C_RaceOth',
 'C_RaceUnk',
 'C_His',
 'C_NonHis',
 'C_EthUnk',
 'GlobalID',
 'C_NEW_PERCT_CHG',
 'D_NEW_PERCT_CHG',
 'C_NEW_7D_MEAN',
 'D_NEW_7D_MEAN',
 'C_NEW_7D_PERCT_CHG',
 'D_NEW_7D_PERCT_CHG']

In [ ]:
dataframe['COUNTY'].value_counts()

BALDWIN       566
UNION         566
COOK          566
CARROLL       566
EFFINGHAM     566
             ... 
FORSYTH       566
GILMER        566
GLYNN         566
GLASCOCK      566
WASHINGTON    566
Name: COUNTY, Length: 162, dtype: int64

In [ ]:
dataframe_counties = dataframe['COUNTY'].value_counts()
dataframe_counties.head(5)

BALDWIN      566
UNION        566
COOK         566
CARROLL      566
EFFINGHAM    566
Name: COUNTY, dtype: int64

# Step 4: Transforming Columns


In [ ]:
dataframe['DATESTAMP']

0        2021/04/13 03:59:59+00
1        2021/02/04 05:00:00+00
2        2021/04/14 03:59:59+00
3        2021/02/05 05:00:00+00
4        2021/04/15 03:59:59+00
                  ...          
91687    2021/07/15 03:59:59+00
91688    2021/07/16 03:59:59+00
91689    2021/07/17 03:59:59+00
91690    2021/07/18 03:59:59+00
91691    2021/07/19 03:59:59+00
Name: DATESTAMP, Length: 91692, dtype: object

In [ ]:
## creating a copy existing columns while keeping the original to provide flexibility
## for analysis

dataframe['DATESTAMP_MOD'] = dataframe['DATESTAMP']
print(dataframe['DATESTAMP_MOD'].head(10))
print(dataframe['DATESTAMP_MOD'].dtypes)


      


0    2021/04/13 03:59:59+00
1    2021/02/04 05:00:00+00
2    2021/04/14 03:59:59+00
3    2021/02/05 05:00:00+00
4    2021/04/15 03:59:59+00
5    2021/02/06 05:00:00+00
6    2021/02/07 05:00:00+00
7    2021/04/16 03:59:59+00
8    2021/04/17 03:59:59+00
9    2021/02/08 05:00:00+00
Name: DATESTAMP_MOD, dtype: object
object


In [ ]:
dataframe['DATESTAMP_MOD'] = pd.to_datetime(dataframe['DATESTAMP_MOD'])
print(dataframe['DATESTAMP_MOD'].dtypes)

datetime64[ns, UTC]


In [ ]:
dataframe[['DATESTAMP', 'DATESTAMP_MOD']]

DATESTAMP             DATESTAMP_MOD
0      2021/04/13 03:59:59+00 2021-04-13 03:59:59+00:00
1      2021/02/04 05:00:00+00 2021-02-04 05:00:00+00:00
2      2021/04/14 03:59:59+00 2021-04-14 03:59:59+00:00
3      2021/02/05 05:00:00+00 2021-02-05 05:00:00+00:00
4      2021/04/15 03:59:59+00 2021-04-15 03:59:59+00:00
...                       ...                       ...
91687  2021/07/15 03:59:59+00 2021-07-15 03:59:59+00:00
91688  2021/07/16 03:59:59+00 2021-07-16 03:59:59+00:00
91689  2021/07/17 03:59:59+00 2021-07-17 03:59:59+00:00
91690  2021/07/18 03:59:59+00 2021-07-18 03:59:59+00:00
91691  2021/07/19 03:59:59+00 2021-07-19 03:59:59+00:00

[91692 rows x 2 columns]

In [182]:
dataframe['DATESTAMP_MOD_DAY'] = dataframe['DATESTAMP_MOD'].dt.date
dataframe['DATESTAMP_MOD_DAY']

0        2021-04-13
1        2021-02-04
2        2021-04-14
3        2021-02-05
4        2021-04-15
            ...    
91687    2021-07-15
91688    2021-07-16
91689    2021-07-17
91690    2021-07-18
91691    2021-07-19
Name: DATESTAMP_MOD_DAY, Length: 91692, dtype: object

In [ ]:
dataframe['DATESTAMP_MOD_YEAR'] = dataframe['DATESTAMP_MOD'].dt.year
dataframe['DATESTAMP_MOD_MONTH'] = dataframe['DATESTAMP_MOD'].dt.month

In [ ]:
dataframe['DATESTAMP_MOD_YEAR']

0        2021
1        2021
2        2021
3        2021
4        2021
         ... 
91687    2021
91688    2021
91689    2021
91690    2021
91691    2021
Name: DATESTAMP_MOD_YEAR, Length: 91692, dtype: int64

In [ ]:
dataframe['DATESTAMP_MOD_MONTH']

0        4
1        2
2        4
3        2
4        4
        ..
91687    7
91688    7
91689    7
91690    7
91691    7
Name: DATESTAMP_MOD_MONTH, Length: 91692, dtype: int64

In [ ]:
dataframe['DATESTAMP_MOD_MONTH_YEAR'] = dataframe['DATESTAMP_MOD'].dt.to_period('M')
dataframe['DATESTAMP_MOD_MONTH_YEAR'].sort_values()

15942    2020-01
26961    2020-01
26960    2020-01
26959    2020-01
26958    2020-01
          ...   
32662    2021-07
32663    2021-07
32664    2021-07
32184    2021-07
91691    2021-07
Name: DATESTAMP_MOD_MONTH_YEAR, Length: 91692, dtype: period[M]

In [181]:
dataframe

OBJECTID   COUNTY  CNTY_FIPS               DATESTAMP  C_New  C_Cum  \
0             1  BALDWIN       1003  2021/04/13 03:59:59+00      4   3795   
1             2  UNKNOWN          1  2021/02/04 05:00:00+00      4   2158   
2             3  BALDWIN       1003  2021/04/14 03:59:59+00      2   3797   
3             4  UNKNOWN          1  2021/02/05 05:00:00+00      1   2159   
4             5  BALDWIN       1003  2021/04/15 03:59:59+00      0   3797   
...         ...      ...        ...                     ...    ...    ...   
91687     91688    WAYNE      49055  2021/07/15 03:59:59+00      0   2809   
91688     91689    WAYNE      49055  2021/07/16 03:59:59+00      0   2809   
91689     91690    WAYNE      49055  2021/07/17 03:59:59+00      0   2809   
91690     91691    WAYNE      49055  2021/07/18 03:59:59+00      0   2809   
91691     91692    WAYNE      49055  2021/07/19 03:59:59+00      0   2809   

       D_New  D_Cum  H_New  H_Cum  ...             DATESTAMP_MOD  \
0          0    115      0    318  ... 2021-04-13 03:59:59+00:00   
1          0      4      0     36  ... 2021-02-04 05:00:00+00:00   
2          0    115      0    318  ... 2021-04-14 03:59:59+00:00   
3          0      4      0     36  ... 2021-02-05 05:00:00+00:00   
4          0    115      0    318  ... 2021-04-15 03:59:59+00:00   
...      ...    ...    ...    ...  ...                       ...   
91687      0     81      0    281  ... 2021-07-15 03:59:59+00:00   
91688      0     81      0    281  ... 2021-07-16 03:59:59+00:00   
91689      0     81      0    281  ... 2021-07-17 03:59:59+00:00   
91690      0     81      0    281  ... 2021-07-18 03:59:59+00:00   
91691      0     81      0    281  ... 2021-07-19 03:59:59+00:00   

       DATESTAMP_MOD_DAY  DATESTAMP_MOD_YEAR  DATESTAMP_MOD_MONTH  \
0             2021-04-13                2021                    4   
1             2021-02-04                2021                    2   
2             2021-04-14                2021                    4   
3             2021-02-05                2021                    2   
4             2021-04-15                2021                    4   
...                  ...                 ...                  ...   
91687         2021-07-15                2021                    7   
91688         2021-07-16                2021                    7   
91689         2021-07-17                2021                    7   
91690         2021-07-18                2021                    7   
91691         2021-07-19                2021                    7   

       DATESTAMP_MOD_MONTH_YEAR  DATESTAMP_MOD_WEEK  DATESTAMP_MOD_QUARTER  \
0                       2021-04                  15                 2021Q2   
1                       2021-02                   5                 2021Q1   
2                       2021-04                  15                 2021Q2   
3                       2021-02                   5                 2021Q1   
4                       2021-04                  15                 2021Q2   
...                         ...                 ...                    ...   
91687                   2021-07                  28                 2021Q3   
91688                   2021-07                  28                 2021Q3   
91689                   2021-07                  28                 2021Q3   
91690                   2021-07                  28                 2021Q3   
91691                   2021-07                  29                 2021Q3   

       DATESTAMP_MOD_WEEK_STRING  DATESTAMP_MOD_DAY_STRING  DATETIME_STRING  
0                             15                2021-04-13          2021-04  
1                              5                2021-02-04          2021-02  
2                             15                2021-04-14          2021-04  
3                              5                2021-02-05          2021-02  
4                             15                2021-04-15          2021-04  
...                          ..

In [ ]:
dataframe['DATESTAMP_MOD_WEEK'] = dataframe['DATESTAMP_MOD'].dt.isocalendar().week
dataframe['DATESTAMP_MOD_WEEK']

0        15
1         5
2        15
3         5
4        15
         ..
91687    28
91688    28
91689    28
91690    28
91691    29
Name: DATESTAMP_MOD_WEEK, Length: 91692, dtype: UInt32

In [ ]:
dataframe['DATESTAMP_MOD_QUARTER'] = dataframe['DATESTAMP_MOD'].dt.to_period('Q')

/usr/local/lib/python3.7/dist-packages/pandas/core/arrays/datetimes.py:1146: UserWarning:

Converting to PeriodArray/Index representation will drop timezone information.



76343    2020Q1
46394    2020Q1
46393    2020Q1
46392    2020Q1
46391    2020Q1
          ...  
32834    2021Q3
32835    2021Q3
32836    2021Q3
33566    2021Q3
91691    2021Q3
Name: DATESTAMP_MOD_QUARTER, Length: 91692, dtype: period[Q-DEC]

In [ ]:
dataframe['DATESTAMP_MOD_QUARTER'].sort_values()

76343    2020Q1
46394    2020Q1
46393    2020Q1
46392    2020Q1
46391    2020Q1
          ...  
32834    2021Q3
32835    2021Q3
32836    2021Q3
33566    2021Q3
91691    2021Q3
Name: DATESTAMP_MOD_QUARTER, Length: 91692, dtype: period[Q-DEC]

In [167]:
dataframe['DATESTAMP_MOD_DAY_STRING'] = dataframe['DATESTAMP_MOD_DAY'].astype(str)
dataframe['DATESTAMP_MOD_WEEK_STRING'] = dataframe['DATESTAMP_MOD_WEEK'].astype(str)
dataframe['DATETIME_STRING'] = dataframe['DATESTAMP_MOD_MONTH_YEAR'].astype(str)

In [ ]:
dataframe

OBJECTID   COUNTY  CNTY_FIPS               DATESTAMP  C_New  C_Cum  \
0             1  BALDWIN       1003  2021/04/13 03:59:59+00      4   3795   
1             2  UNKNOWN          1  2021/02/04 05:00:00+00      4   2158   
2             3  BALDWIN       1003  2021/04/14 03:59:59+00      2   3797   
3             4  UNKNOWN          1  2021/02/05 05:00:00+00      1   2159   
4             5  BALDWIN       1003  2021/04/15 03:59:59+00      0   3797   
...         ...      ...        ...                     ...    ...    ...   
91687     91688    WAYNE      49055  2021/07/15 03:59:59+00      0   2809   
91688     91689    WAYNE      49055  2021/07/16 03:59:59+00      0   2809   
91689     91690    WAYNE      49055  2021/07/17 03:59:59+00      0   2809   
91690     91691    WAYNE      49055  2021/07/18 03:59:59+00      0   2809   
91691     91692    WAYNE      49055  2021/07/19 03:59:59+00      0   2809   

       D_New  D_Cum  H_New  H_Cum  ...             DATESTAMP_MOD  \
0          0    115      0    318  ... 2021-04-13 03:59:59+00:00   
1          0      4      0     36  ... 2021-02-04 05:00:00+00:00   
2          0    115      0    318  ... 2021-04-14 03:59:59+00:00   
3          0      4      0     36  ... 2021-02-05 05:00:00+00:00   
4          0    115      0    318  ... 2021-04-15 03:59:59+00:00   
...      ...    ...    ...    ...  ...                       ...   
91687      0     81      0    281  ... 2021-07-15 03:59:59+00:00   
91688      0     81      0    281  ... 2021-07-16 03:59:59+00:00   
91689      0     81      0    281  ... 2021-07-17 03:59:59+00:00   
91690      0     81      0    281  ... 2021-07-18 03:59:59+00:00   
91691      0     81      0    281  ... 2021-07-19 03:59:59+00:00   

       DATESTAMP_MOD_DAY  DATESTAMP_MOD_YEAR  DATESTAMP_MOD_MONTH  \
0             2021-04-13                2021                    4   
1             2021-02-04                2021                    2   
2             2021-04-14                2021                    4   
3             2021-02-05                2021                    2   
4             2021-04-15                2021                    4   
...                  ...                 ...                  ...   
91687         2021-07-15                2021                    7   
91688         2021-07-16                2021                    7   
91689         2021-07-17                2021                    7   
91690         2021-07-18                2021                    7   
91691         2021-07-19                2021                    7   

       DATESTAMP_MOD_MONTH_YEAR  DATESTAMP_MOD_WEEK  DATESTAMP_MOD_QUARTER  \
0                       2021-04                  15                 2021Q2   
1                       2021-02                   5                 2021Q1   
2                       2021-04                  15                 2021Q2   
3                       2021-02                   5                 2021Q1   
4                       2021-04                  15                 2021Q2   
...                         ...                 ...                    ...   
91687                   2021-07                  28                 2021Q3   
91688                   2021-07                  28                 2021Q3   
91689                   2021-07                  28                 2021Q3   
91690                   2021-07                  28                 2021Q3   
91691                   2021-07                  29                 2021Q3   

       DATESTAMP_MOD_WEEK_STRING  DATESTAMP_MOD_DAY_STRING  DATETIME_STRING  
0                             15                2021-04-13          2021-04  
1                              5                2021-02-04          2021-02  
2                             15                2021-04-14          2021-04  
3                              5                2021-02-05          2021-02  
4                             15                2021-04-15          2021-04  
...                          ..

# Step 5: Getting required counties for analysis
Counties to be analyzed include the following:
- Cobb
- DeKalb
- Fulton
- Gwinnett
- Hall



In [ ]:
dataframe['COUNTY']

0        BALDWIN
1        UNKNOWN
2        BALDWIN
3        UNKNOWN
4        BALDWIN
          ...   
91687      WAYNE
91688      WAYNE
91689      WAYNE
91690      WAYNE
91691      WAYNE
Name: COUNTY, Length: 91692, dtype: object

In [ ]:
countList = ['COBB', 'DEKALB', 'FULTON', 'GWINNETT', 'HALL']
countList

['COBB', 'DEKALB', 'FULTON', 'GWINNETT', 'HALL']

In [ ]:
selectCounties = dataframe[dataframe['COUNTY'].isin(countList)]
len(selectCounties)

2830

## Getting specific date/time frame needed


dataframe = leng ~90,000 rows
selectCounties = length 2830
selectCountyTime = ???

In [ ]:
selectCountyTime = selectCounties

In [ ]:
selectCountyTime['DATESTAMP_MOD_MONTH_YEAR']

6056     2020-08
6414     2020-08
6417     2020-09
6422     2020-09
6823     2020-09
          ...   
79995    2020-05
79996    2020-05
79997    2020-05
79998    2020-05
79999    2020-05
Name: DATESTAMP_MOD_MONTH_YEAR, Length: 2830, dtype: period[M]

In [176]:
selectCountTime_april2020 = selectCountyTime[selectCountyTime['DATESTAMP_MOD_MONTH_YEAR'] == '2020-04']
len(selectCountTime_april2020)


150

In [162]:
selectCountTime_aprilmay2020 = selectCountyTime[(selectCountyTime['DATESTAMP_MOD_MONTH_YEAR'] == '2020-05') | (selectCountyTime['DATESTAMP_MOD_MONTH_YEAR'] == '2020-04')]
len(selectCountTime_aprilmay2020)

305

In [177]:
selectCountTime_aprilmay2020.head(50)

OBJECTID  COUNTY  CNTY_FIPS               DATESTAMP  C_New  C_Cum  \
7464      7465  FULTON       5049  2020/04/01 03:59:59+00     97   1499   
7465      7466  FULTON       5049  2020/04/02 03:59:59+00     50   1549   
7466      7467  FULTON       5049  2020/04/03 03:59:59+00     68   1617   
7467      7468  FULTON       5049  2020/04/04 03:59:59+00     38   1655   
7468      7469  FULTON       5049  2020/04/05 03:59:59+00     30   1685   
7469      7470  FULTON       5049  2020/04/06 03:59:59+00     62   1747   
7470      7471  FULTON       5049  2020/04/07 03:59:59+00     77   1824   
7471      7472  FULTON       5049  2020/04/08 03:59:59+00     69   1893   
7472      7473  FULTON       5049  2020/04/09 03:59:59+00     59   1952   
7473      7474  FULTON       5049  2020/04/10 03:59:59+00     55   2007   
7474      7475  FULTON       5049  2020/04/11 03:59:59+00     53   2060   
7475      7476  FULTON       5049  2020/04/12 03:59:59+00     39   2099   
7476      7477  FULTON       5049  2020/04/13 03:59:59+00     54   2153   
7477      7478  FULTON       5049  2020/04/14 03:59:59+00     60   2213   
7478      7479  FULTON       5049  2020/04/15 03:59:59+00     83   2296   
7479      7480  FULTON       5049  2020/04/16 03:59:59+00     57   2353   
7480      7481  FULTON       5049  2020/04/17 03:59:59+00     67   2420   
7481      7482  FULTON       5049  2020/04/18 03:59:59+00     39   2459   
7482      7483  FULTON       5049  2020/04/19 03:59:59+00     73   2532   
7483      7484  FULTON       5049  2020/04/20 03:59:59+00     84   2616   
7484      7485  FULTON       5049  2020/04/21 03:59:59+00     60   2676   
7485      7486  FULTON       5049  2020/04/22 03:59:59+00     59   2735   
7486      7487  FULTON       5049  2020/04/23 03:59:59+00     35   2770   
7487      7488  FULTON       5049  2020/04/24 03:59:59+00     65   2835   
7488      7489  FULTON       5049  2020/04/25 03:59:59+00     39   2874   
7489      7490  FULTON       5049  2020/04/26 03:59:59+00     36   2910   
7490      7491  FULTON       5049  2020/04/27 03:59:59+00     51   2961   
7491      7492  FULTON       5049  2020/04/28 03:59:59+00     77   3038   
7492      7493  FULTON       5049  2020/04/29 03:59:59+00     78   3116   
7493      7494  FULTON       5049  2020/04/30 03:59:59+00     73   3189   
7494      7495  FULTON       5049  2020/05/01 03:59:59+00     54   3243   
7495      7496  FULTON       5049  2020/05/02 03:59:59+00     29   3272   
7496      7497  FULTON       5049  2020/05/03 03:59:59+00     33   3305   
7497      7498  FULTON       5049  2020/05/04 03:59:59+00     47   3352   
7498      7499  FULTON       5049  2020/05/05 03:59:59+00     60   3412   
7499      7500  FULTON       5049  2020/05/06 03:59:59+00     58   3470   
7500      7501  FULTON       5049  2020/05/07 03:59:59+00     75   3545   
7501      7502  FULTON       5049  2020/05/08 03:59:59+00     47   3592   
7502      7503  FULTON       5049  2020/05/09 03:59:59+00     20   3612   
7503      7504  FULTON       5049  2020/05/10 03:59:59+00     31   3643   
7504      7505  FULTON       5049  2020/05/11 03:59:59+00     75   3718   
7505      7506  FULTON       5049  2020/05/12 03:59:59+00     50   3768   
7506      7507  FULTON       5049  2020/05/13 03:59:59+00     55   3823   
7507      7508  FULTON       5049  2020/05/14 03:59:59+00     76   3899   
7508      7509  FULTON       5049  2020/05/15 03:59:59+00     63   3962   
7509      7510  FULTON       5049  2020/05/16 03:59:59+00     30   3992   
7510      7511  FULTON       5049  2020/05/17 03:59:59+00     24   4016   
7511      7512  FULTON       5049  2020/05/18 03:59:59+00     73   4089   
7512      7513  FULTON       5049  2020/05/19 03:59:59+00     51   4140   
7513      7514  FULTON       5049  2020/05/20 03:59:59+00     36   4176   

      D_New  D_Cum  H_New  H_Cum  ...  C_NEW_7D_PERCT_CHG  D_NEW_7D_PERCT_CHG  \
7464     10    109     22    460  ...            0.076923            0.214286   
7465      3    112

## Step 6: Creating the final Dataframe for visualization

In [180]:
finalDF = selectCountTime_aprilmay2020[[
    'COUNTY', 
    'DATESTAMP_MOD', 
    'DATESTAMP_MOD_DAY', 
    'DATESTAMP_MOD_DAY_STRING', 
    'DATETIME_STRING', 
    'DATESTAMP_MOD_MONTH_YEAR'
]]

KeyError: ignored